custom_components/tuya_local/devices/kogan_kablfan1csa_fan.yaml

```diff
@@ -0,0 +1,109 @@
+name: Kogan Smarterhome Bladeless Purifier Fan
+products:
+  - id: kablfan1csa
```

Comment:
Are these product ids actually matching the catalogue part numbers?

The id here should be from the Tuya protocol (broadcast as part of the discovery protocol, as reported by tinytuya, or from the device info related cloud API functions (generally the same functions that can tell you the local key).